membersdir.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<head>
+    <title>ObieSource</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/style.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+</head>
+<body>
+    <div class="main">
+        <a href="/">Go Home</a>
+        <h2>Obiesource Members Directory</h2>
+        <ul>
+            <h3 class="diroff">Club Officers</h3>
+            <h3 class="dirmem">Members</h3>
+            <div class="dirend"></div>
+        </ul>
```

Comment:
It's probably good to avoid putting stuff other that `<li>` tags inside an `<ul>` tag. Browsers are forgiving, but this technically breaks the [HTML specification](https://html.spec.whatwg.org/multipage/grouping-content.html#the-ul-element:concept-element-content-model).